home.js

```diff
@@ -0,0 +1,149 @@
+const nav = document.querySelector('.nav-items');
+const humburger = document.querySelector('.humburger');
+humburger.addEventListener('click', () => {
+  humburger.classList.toggle('active');
+  document.body.style.overflow = 'hidden';
+  nav.classList.toggle('active');
+});
+document.querySelectorAll('.nav-item').forEach((n) => n.addEventListener('click', () => {
+  humburger.classList.remove('active');
+  document.body.style.overflow = 'auto';
+  nav.classList.remove('active');
+}));
+
+const activities = document.querySelector('.featured-activities');
+const activitiesArray = [
+  {
+    id: '1',
+    image: 'images/speaker_01.png',
+    tittle: 'Remembering the Children:',
+    place: 'ew Iceland Heritage Museum & Independent',
+    city: 'Gimli, MB',
+  },
+
+  {
+    id: '2',
+    image: 'images/speaker_02.png',
+    tittle: 'Nuit Blanche Kids Zone at ',
+    place: 'Manitoba Childrens Museum',
+    city: 'Winnipeg, MB',
+  },
+
+  {
+    id: '3',
+    image: 'images/speaker_03.png',
+    tittle: 'WSO Community Celebration ',
+    place: 'Centennial Concert Hall',
+    city: 'Winnipeg, MB',
+  },
+
+  {
+    id: '4',
+    image: 'images/speaker_04.png',
+    tittle: 'Nuit Blanche @ WAG',
+    place: 'WAG-Qaumajuq',
+    city: 'Winnipeg, MB',
+  },
+
+  {
+    id: '5',
+    image: 'images/speaker_05.png',
+    tittle: 'Kidz Culture',
+    place: 'Creekside Park',
+    city: 'Flin Flon, MB',
```

Comment:
Kindly note that each individual speaker should have title, subtitle and a description of the speaker. Your design is missing the description. Please update it to match the design :pray: 


 Your design            |  The  design 
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-30 07-25-07](https://user-images.githubusercontent.com/65233860/204733987-d82e3be9-539e-4b51-82cb-75484abd417e.png)  |  ![Screenshot from 2022-11-30 07-25-17](https://user-images.githubusercontent.com/65233860/204733999-4e21b8a9-53b3-4ddd-aa19-081a92e21435.png)